db/main.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"database/sql"
+
+	"github.com/cyverse-de/dbutil"
+	"github.com/pkg/errors"
+)
+
+// InitDatabase establishes a database connection and verifies that the database can be reached.
+func InitDatabase(driverName, databaseURI string) (*sql.DB, error) {
+	wrapMsg := "unable to initialize the database"
+
+	// Create a database connector to establish the connection for us.
+	connector, err := dbutil.NewDefaultConnector("1m")
+	if err != nil {
+		return nil, errors.Wrap(err, wrapMsg)
+	}
+
+	// Establish the database connection.
+	db, err := connector.Connect(driverName, databaseURI)
+	if err != nil {
+		return nil, errors.Wrap(err, wrapMsg)
+	}
+
+	// Verify that we can ping the database.
```

Comment:
`connector.Connect` already pings the DB, so probably this section should be dropped.

Reply:
I'd forgotten that `connector.Connect` already pings the database. I'll remove this.